src/components/network/network-explorer.js

```diff
@@ -107,6 +107,16 @@ const NetworkExplorer = ({
     const [filteredVoltageLevels, setFilteredVoltageLevels] = React.useState(
         []
     );
+    const [substationsLoaded, setSubstationsLoaded] = React.useState(false);
+
+    useEffect(() => {
```

Comment:
I think we should simplify the API. Maybe create our own hook useNetwork or something ? I'm not sure. It's a pain to have to deal with the loading everywhere.